ccd-definition/ComplexTypes/SmallClaimsHearing-SDO-nonprod.json

```diff
@@ -16,11 +16,30 @@
   {
     "ID": "SmallClaimsHearing",
     "ListElementCode": "time",
-    "FieldType": "FixedList",
+    "FieldType": "FixedRadioList",
     "FieldTypeParameter": "SmallClaimsTimeEstimate",
-    "ElementLabel": "Select time estimate",
+    "ElementLabel": " ",
     "SecurityClassification": "Public"
   },
+  {
+    "ID": "SmallClaimsHearing",
+    "ListElementCode": "otherHours",
+    "ElementLabel": "Hour(s)",
+    "FieldType": "Number",
+    "RegularExpression": "\\d",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "time = \"OTHER\""
+  },
+  {
+    "ID": "SmallClaimsHearing",
+    "ListElementCode": "otherMinutes",
+    "ElementLabel": "Minute(s)",
+    "FieldType": "Number",
+    "Max": 59,
+    "RegularExpression": "\\d",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "time = \"OTHER\""
```

Comment:
I feel that we should move these changes into a H&L-SDO-nonprod files to keep them sepeate from the SDO teams changes